core/jobs_registry.py

```diff
@@ -49,6 +49,7 @@
     exp_jobs_one_off.ExplorationContentValidationJobForCKEditor,
     exp_jobs_one_off.ExplorationMigrationValidationJobForCKEditor,
     exp_jobs_one_off.ImageDataMigrationJob,
+    exp_jobs_one_off.CustomizationArgsValidationJob,
```

Comment:
InteractionCustomizationArgsValidationJob

(we use customization args in other places as well)

Reply:
Done